packages/eas-cli/src/submissions/ios/AppSpecificPasswordSource.ts

```diff
@@ -0,0 +1,54 @@
+import chalk from 'chalk';
+import wordwrap from 'wordwrap';
+
+import log from '../../log';
+import { promptAsync } from '../../prompts';
+
+export enum AppSpecificPasswordSourceType {
+  userDefined,
+  prompt,
+}
+
+interface AppSpecificPasswordSourceBase {
+  sourceType: AppSpecificPasswordSourceType;
+}
+
+interface AppSpecificPasswordUserDefinedSource extends AppSpecificPasswordSourceBase {
+  sourceType: AppSpecificPasswordSourceType.userDefined;
+  appSpecificPassword: string;
+}
+
+interface AppSpecificPasswordPromptSource extends AppSpecificPasswordSourceBase {
+  sourceType: AppSpecificPasswordSourceType.prompt;
+}
+
+export type AppSpecificPasswordSource =
+  | AppSpecificPasswordUserDefinedSource
+  | AppSpecificPasswordPromptSource;
+
+export async function getAppSpecificPasswordAsync(source: AppSpecificPasswordSource) {
+  if (source.sourceType === AppSpecificPasswordSourceType.userDefined) {
+    return source.appSpecificPassword;
+  } else if (source.sourceType === AppSpecificPasswordSourceType.prompt) {
+    log.addNewLineIfNone();
+    const wrap = wordwrap(process.stdout.columns || 80);
+    log(
+      wrap(
+        chalk.bold(
+          `The password is only used to upload the build to Apple TestFlight and never stored on Expo servers`
+        )
+      )
+    );
+
+    const { appSpecificPassword } = await promptAsync({
+      name: 'appSpecificPassword',
+      message: 'Your Apple application-specific password:',
```

Comment:
```suggestion
      message: 'Your Apple app-specific password:',
```

<img width="427" alt="Screenshot 2020-11-04 at 11 06 19" src="https://user-images.githubusercontent.com/5256730/98097793-c89ebb80-1e8d-11eb-8f9d-3cfd2cccbaf1.png">

We should be consistent with Apple.

Reply:
+1 for this. I was consistent with Fastlane until now, but was annoyed with very long names, e.g. `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD`